nekoyume/Assets/_Scripts/UI/Widget/Popup/BuffBonusPopup.cs

```diff
@@ -151,5 +173,44 @@ private void PushAction(bool advanced)
             ActionManager.Instance.HackAndSlashRandomBuff(advanced).Subscribe();
             Close();
         }
+
+        private void ShowBuffListView()
+        {
+            buffListView.SetActive(true);
+            UpdateBuffListView();
+        }
+
+        private void UpdateBuffListView()
+        {
+            var randomBuffsheet = Game.Game.instance.TableSheets.CrystalRandomBuffSheet;
+            var randomBuffGroupsByRank = randomBuffsheet.Select(line => line.Value).GroupBy(row => row.Rank);
+
+            InitializeChildren();
+
+            foreach (var randomBuffGroup in randomBuffGroupsByRank)
+            {
+                // Grade
+                var go = Instantiate(titlePrefab, cellContainer);
+                go.GetComponent<BuffBonusTitleCell>().Set(randomBuffGroup.Key);
+
+                foreach (var randomBuffRow in randomBuffGroup)
+                {
+                    // buff
+                    go = Instantiate(buffPrefab, cellContainer);
+                    go.GetComponent<BuffBonusBuffCell>().Set(randomBuffRow);
+                }
+            }
+        }
+
+        private void InitializeChildren()
```

Comment:
Do you can explain about this method?
Sorry but I can't understand the relationship between 'Initializing' and 'Destruction'...
What is this destruction for?

Reply:
The part that I forgot to modify when transferred the previous. I think it's better not to separate the method.

Reply:
How about this?

Reply:
Um... I still don't know why it destroy cells.